vspc/server.py

```diff
@@ -143,6 +143,12 @@ def handle_vmotion_complete(self, socket, data):
         peer = socket.getpeername()
         LOG.debug("<< %s VMOTION-COMPLETE %s", peer, data)
 
+    def handle_vmotion_abort(self, writer, data):
+        socket = writer.get_extra_info('socket')
+        peer = socket.getpeername()
+        LOG.debug("<< %s VMOTION-ABORT %s", peer, data)
+        writer.close()
```

Comment:
Do we have to drain the socket, too, or is closing enough? Is that maybe a source for our memory-leak, that we don't drain the sockets before closing them (in other places)?

Reply:
I've checked if that can cause the memory leak, but it doesn't... need to wrap the entire code with `tracemalloc` and check that maybe we have living coroutines. Also, need to check if bumping the python version to at least 3.8 can solve the issue or/and use timeouts on tcp/tls connections.

Reply:
Do we close the socket here? Because your updated commit-message tells us, the source-host sends this command and the connection should be kept alive as it's the one to be used in the future.

Reply:
Hmmm... seems like yes, we don't need to close connection here.